### PR TITLE
Instead of doubling the chance of the gravity event happening every time its triggered, it only doubles it if the gravity generator is active when the event ends

### PR DESCRIPTION
**Pull request by alexkar598**
# General Documentation

### Intent of your Pull Request

Discourages rebooting the gravity generator, bypassing the event completly

### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Instead of doubling the chance of the gravity event happening every time its triggered, it only doubles it if the gravity generator is active when the event ends

### What should players be aware of when it comes to the changes your PR is implementing?
Instead of doubling the chance of the gravity event happening every time its triggered, it only doubles it if the gravity generator is active when the event ends

### What general grouping does this PR fall under? 
Events

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
No

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
For example, "This new chem will heal X brute damage". 
Doubling = original value * 2
Alternatively if youre feeling spicy, you can do original value / 0.5

# Changelog
:cl:  
tweak: Instead of doubling the chance of the gravity event happening every time its triggered, it only doubles it if the gravity generator is active when the event ends
/:cl:

### CODE REVIEW

**Comment by Cartlord** [comment]
My main issue with this is that there likely won't be anything pointing to this in-game, so people will reboot the GravGen anyways and get screwed for it.

**Review by JamieD1 — APPROVED** [review]
